cnd/src/http_api.rs

```diff
@@ -141,19 +135,6 @@ impl Serialize for Http<Chain> {
 
 impl_serialize_type_with_fields!(htlc_location::Bitcoin { "txid" => txid, "vout" => vout });
 impl_serialize_http!(crate::ethereum::Address);
```

Comment:
I think this too.

Probably the whole macro as-well :tada:

Reply:
Nice.

Reply:
Inlined this one.